app/policies/playlist_activity_policy.rb

```diff
@@ -0,0 +1,3 @@
+class PlaylistActivityPolicy < StudentManagementPolicy
+  alias_method :new?, :index?
```

Comment:
policy for managing a student: is the teacher in the same grove as the student, OR is the teach an admin in the same school as the student? I think that should suffice, but lmk if I'm overlooking a use case.